p2p/__init__.py

```diff
@@ -602,44 +602,49 @@ def clone_content_item(self, slug, clone_slug, keep_embeds=False, keep_relateds=
                 }
                 payload['related_items'].append(related_item)
 
+        found_contributors = find_clone_contributors(content_item)
+
+        if 'contributors' in content_item:
```

Comment:
But again, it's more this ` payload['contributors'] = content_item['contributors']` that I have a question on.  I just don't think the payload accepts the contributors list in the same format that it spits it out in.